### ISSUE TITLE
Add requirements.txt 

### ISSUE THREAD

**Issue by AbyAbraham21** [simvue-io/integrations#13]
Trying to clone the integrations repo to run an FDS example leads to the following error and user having to install it. ModuleNotFoundError: No module named 'f90nml'

Will be friendly to have a requirements.txt file with dependencies

**Comment by wk9874** [comment]
@AbyAbraham21 hopefully you can do `pip install git+https://github.com/simvue-io/integrations.git@main#egg=simvue-integrations[fds]` to install integrations and required modules for FDS. Eventually once integrations has a PyPI release, you'd have to do `pip install simvue-integrations -E fds`

**Comment by alahiff** [comment]
I just encountered this. We should probably try to have this sorted out in time for the trial.

**Comment by AbyAbraham21** [comment]
Yes, agreed. Will have it done as part of the review. Thanks for flagging @alahiff

**Comment by wk9874** [comment]
This will be fixed now that FDS has its own package :)
